### ISSUE TITLE
Update sandbox by re-upload of swagger

### ISSUE THREAD

**Issue by mattfrear** [getsandbox/feedback#2]
It would be nice to be able to update an existing sandbox by re-uploading the swagger. At the moment I need to delete my sandbox and upload again with the same name.

**Comment by nhoughto** [comment]
Agreed, v1 of this might be a destructive re-import, longer term / v2 looking to do a merge with existing codebase. On the list!

**Comment by nhoughto** [comment]
This is progressing, just need to finish off the tests and its good to go, should be this week.

**Comment by mattfrear** [comment]
Thanks Nick :smile:
